flink-runtime/src/main/java/org/apache/flink/runtime/scheduler/strategy/SchedulingStrategy.java

```diff
@@ -55,4 +55,9 @@ public interface SchedulingStrategy {
      * @param resultPartitionId The id of the result partition
      */
     void onPartitionConsumable(IntermediateResultPartitionID resultPartitionId);
+
+    /** Called when the batch job recovery is finished. */
+    default void scheduleVerticesAfterRecovering() {
```

Comment:
I prefer to name it as `scheduleAllVerticesIfPossible`. And comment that it schedules all vertices except for finished vertices and vertices whose inputs are not ready.
Job recovery just leverages its capability to resume job scheduling.